vortex-array/src/array/varbinview/mod.rs

```diff
@@ -31,13 +31,12 @@ mod variants;
 
 #[derive(Clone, Copy, Debug)]
 #[repr(C, align(8))]
-struct Inlined {
+pub struct Inlined {
```

Comment:
Why have these become pub?

Reply:
so I didn't need to allow dead_code on their constructors (in case we want to use them later). also, the BinaryView union type is public but these aren't, which is... weird?